### PR TITLE
Refactor/sidebar sass

### PR DESCRIPTION
**Pull request by Ronsho**
* Added Sass support to the project (linting + webpack)
* refactored sidebar.css to sidebar.scss

### CODE REVIEW

File: packages/devtools-launchpad/src/components/sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
+$sidebar-width: 200px;
```
Comment (jasonLaster) [inline packages/devtools-launchpad/src/components/sidebar.scss]: can we remove `sidebar.css`? Can we rename `sidebar.scss` to `sidebar.css`? It'd be nice not to differentiate between the two

Comment (Ronsho) [reply packages/devtools-launchpad/src/components/sidebar.scss]: Removed the sidebar.css. Don't think it's common to name give a .css suffix to a sass file...

**Comment by wldcordeiro** [comment]
Hadn't we started using PostCSS in this or the Debugger? I think it'd be nice if we could continue using it over Sass or maybe a CSS-in-JS solution.

**Comment by nyrosmith** [comment]
I think a css-in-js solution would be really cool because it would give us the best of both worlds

**Comment by jasonLaster** [comment]
I agree, it'd be neat to try css-modules and glamorous. Lets make separate branches for each to see what it would look like

**Comment by digitarald** [comment]
@jasonLaster can this be closed for now or and file any follow up work?

**Pull request closed without merging**